registry/monitoring.go

```diff
@@ -77,26 +78,22 @@ func NewInstrumentedClient(next Client) Client {
 	}
 }
 
-func (m *instrumentedClient) Manifest(id image.Ref) (res image.Info, err error) {
+func (m *instrumentedClient) Manifest(ctx context.Context, ref string) (res image.Info, err error) {
 	start := time.Now()
-	res, err = m.next.Manifest(id)
+	res, err = m.next.Manifest(ctx, ref)
 	remoteDuration.With(
 		LabelRequestKind, RequestKindMetadata,
 		fluxmetrics.LabelSuccess, strconv.FormatBool(err == nil),
 	).Observe(time.Since(start).Seconds())
 	return
 }
 
-func (m *instrumentedClient) Tags(id image.Name) (res []string, err error) {
+func (m *instrumentedClient) Tags(ctx context.Context) (res []string, err error) {
```

Comment:
On line 69 (I cannot add a comment there, so doing here), there is a type declaration for InstrumentedClient. I cannot see it used anywhere. If I am right, can the declaration be removed, please?

Reply:
Removed that needless declaration, and the similar one above it.